test/api/search_controller_test.rb

```diff
@@ -430,4 +430,101 @@ class SearchControllerTest < ActionDispatch::IntegrationTest
 
     puts "#{File.basename(__FILE__)}: #{self.method_name} successful!"
   end
+
+  sample_disease_facet = SearchFacet.new(identifier: 'disease',
```

Comment:
Is there a reason why you are declaring these facets outside of the test or setup block?  Since we have facets already being created in `db/seeds.rb`, and real data going into BigQuery from upstream integration tests, shouldn't we leverage those instead?  I'm all for a self-contained test, but this seems strange to me.

Reply:
yeah, I see how weird this looks.  the goal was self-contained unit tests, (which is why I took the time to refactor generate_bq_query_string so that it took the actual db facet rather than an id as part of its parameters).  Then I realized all four of the unit tests wanted the same facets, so I pulled them out of the indvidiual tests.  It doesn't feel good to me to have tests this low-level rely on db/seed, since these are just validating a single method with exact string comparison asserts.  Maybe group the tests in a block/module?  I'm open to suggestions

Reply:
Yeah, maybe take these tests out and make them their own module/file, and then have those facets declared in the `setup` block.  That would alleviate my concerns over the construction of the tests.

Reply:
cool, I pulled them into a separate file so they can have a dedicated setup method -- agree, it looks better this way